src/elements/kw-auth-modal/kw-auth-modal.html

```diff
@@ -146,6 +146,7 @@
                 })
                     .then(r => {
                         if (r.ok) {
+                            this.loading = false;
```

Comment:
I think this line should be outside of the if. If there is an error, we still want to reset the loading state